sdk/src/types.ts

```
@@ -52,7 +56,8 @@ export type ChainConfig = {
 
 export type WormholeConfig = {
   env: Environment;
-  rpcs: Rpcs;
+  rpcs: ChainResourceMap;
+  rest: ChainResourceMap;
```

Should it be suggested that an integrator provide a rest endpoint for Sei? (Like we do for RPCs?)

---

You are right. I think that PR was merged after I rebased. I'll do it again and add the warning for rest resources as well.

---

There's an rpc validation method that throws an error in the console if the integrator doesn't provide an rpc for a given network.  I think it should throw an error if this isn't provided for sei

---

Refactored and implemented for REST and Sei